src/static_files.rs

```diff
@@ -83,8 +83,6 @@ pub struct HandleOpts<'a> {
 pub struct StaticFileResponse {
     /// Inner HTTP response.
     pub resp: Response<Body>,
-    /// If the inner HTTP response is already pre-compressed.
-    pub is_precompressed: bool,
```

Comment:
> There is a merge conflict with https://github.com/static-web-server/static-web-server/pull/368 here unfortunately. Also, removing StaticFileResponse::is_precompressed field is a breaking change. Let me know if you’d prefer me to keep it despite being unused.

Yes, this is a breaking change. But it's also fine to supersede it by checking the `Content-Encoding` instead. However, this has to be mentioned in the next release.